src/wp-includes/media.php

```diff
@@ -2070,20 +2095,14 @@ function wp_img_tag_add_decoding_attr( $image, $context ) {
 		return $image;
 	}
 
-	/**
-	 * Filters the `decoding` attribute value to add to an image. Default `async`.
-	 *
-	 * Returning a falsey value will omit the attribute.
-	 *
-	 * @since 6.1.0
-	 *
-	 * @param string|false|null $value   The `decoding` attribute value. Returning a falsey value
-	 *                                   will result in the attribute being omitted for the image.
-	 *                                   Otherwise, it may be: 'async' (default), 'sync', or 'auto'.
-	 * @param string            $image   The HTML `img` tag to be filtered.
-	 * @param string            $context Additional context about how the function was called
-	 *                                   or where the img tag is.
+	/*
+	 * Only apply the decoding attribute to images that don't already have a decoding attribute.
 	 */
```

Comment:
```suggestion
	// Only apply the decoding attribute to images that don't already have a decoding attribute.
```

Use single-line comment format instead of multi-line for single line docblock.